ext/wddx/wddx.c

```diff
@@ -877,7 +877,7 @@ static void php_wddx_pop_element(void *user_data, const XML_Char *name)
 		!strcmp((char *)name, EL_DATETIME)) {
 		wddx_stack_top(stack, (void**)&ent1);
 
-		if (!ent1->data) {
+		if (Z_TYPE(ent1->data) == IS_UNDEF) {
```

Comment:
I am not sure what's this come from,   there already is Z_TYPE(ent1->data) == IS_UNDEF in master